gensrc/proto/persistent_index.proto

```diff
@@ -70,4 +70,5 @@ message PersistentIndexMetaPB {
     // only store a version to get file name
     EditVersionPB l1_version = 5;
     uint32 format_version = 6;
+    uint64 usage =7;
```

Comment:
What's the usage of this variable?

Reply:
> What's the usage of this variable?

`_usage` is the total data size of PersistentIndex. Because the some keys are varlen, so i can't use `_size` to calculate the data size.

Reply:
Could we unify the _size and _usage?
What's the different cases from them?

Reply:
Another question, where the _usage is used?

Reply:
> Another question, where the _usage is used?

`_size` mean how many kvs exist in persistent index and `_usage` means the actual data size of all kvs in persistent index. `_size` and `_usage` are used to estimate the shard num, page num and bucket num in `_merge_compaction`.

We can get a almost accurate solution according to the `_size` and `_usage`, so we can plan the data distribution in new l1 file and minimize the data movement.